extensions/wikidata/src/org/openrefine/wikidata/qa/ConstraintFetcher.java

```diff
@@ -134,4 +135,35 @@
      * Can this property be used on items?
      */
     boolean usableOnItems(PropertyIdValue pid);
+
+    /**
+     * Retrieves the minimum difference value of the range
```

Comment:
It would be good to mention that this is used by the difference between range constraint (otherwise one might think this is for another constraint).

Reply:
Sure.